packages/marshal/src/passStyleOf.js

```diff
@@ -269,7 +269,7 @@ export function assertCanBeRemotable(val) {
 }
 
 /**
- * @param {Remotable} val
+ * @param {Remotable<*>} val
```

Comment:
```suggestion
 * @template T
 * @param {any} val
 * @returns {asserts is Remotable<T>}
```

Reply:
I'll get back to passStyleOf.js after you merge #2460

Reply:
got back as promised ;)